components/mongodb/actions/search-documents/search-documents.mjs

```diff
@@ -0,0 +1,34 @@
+import mongodbApp from "../../mongodb.app.mjs";
+
+export default {
+  key: "mongodb-search-documents",
+  name: "Search Documents",
+  description: "Search for specific documents or return all documents. [See the docs here](https://docs.mongodb.com/manual/reference/method/db.collection.find/)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    mongodbApp,
+    collection: {
+      propDefinition: [
+        mongodbApp,
+        "collection",
+      ],
+      description: "Collection containing the document to search for.",
+    },
+    filter: {
+      label: "Filter",
+      description: "Object to use as a filter",
+      type: "object",
+      optional: true,
+    },
```

Comment:
I noticed that all of the search values are converted to strings. I think this component should include optional props `parseNumbers`, `parseBooleans`, and `parseDates` like in create-new-document and update-a-document.
```suggestion
    filter: {
      label: "Filter",
      description: "Object to use as a filter. Dates must follow `yyyy-mm-dd` format.",
      type: "object",
      optional: true,
    },
    parseNumbers: {
      propDefinition: [
        mongodbApp,
        "parseNumbers",
      ],
      optional: true,
    },
    parseBooleans: {
      propDefinition: [
        mongodbApp,
        "parseBooleans",
      ],
      optional: true,
    },
    parseDates: {
      propDefinition: [
        mongodbApp,
        "parseDates",
      ],
      optional: true,
    },
```

Reply:
Hey, It's working properly for me, but indeed, it's a good idea to add these params here. Thanks!